src/renderer/pages/Staking/Operations/Bond/Bond.tsx

```diff
@@ -146,73 +144,34 @@ export const Bond = () => {
     const transactions = getBondTxs(Object.keys(validators));
 
     if (signer && isMultisig(txAccounts[0])) {
-      const multisigTx = getMultisigTx(txAccounts[0], signer.accountId, transactions[0]);
-      setMultisigTx(multisigTx);
+      const wrapAsMulti = {
```

Comment:
It is not actually a Wrapper. It is just a type that contain parameters for concrete wrapper. The Wrapper is just a function (can be carried if it needs additional parameters) that accepts Transaction and return Transaction